src/AccountInfoClient.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+
+namespace JVDS\YukiClient;
+
+use JVDS\YukiClient\Exception\Exception;
+use JVDS\YukiClient\Exception\UnexpectedTypeException;
+use JVDS\YukiClient\Soap\ClientFactory;
+use SimpleXMLElement;
+
+class AccountInfoClient extends Client
+{
+    private const WSDL = 'https://api.yukiworks.nl/ws/AccountingInfo.asmx?wsdl';
+
+    protected const SESSION_ID_ARG = 'sessionID';
+
+    public function __construct(
+        string $accessKey,
+        array $soapClientOptions = [],
+        ClientFactory $factory = null,
+        string $wsdl = self::WSDL
+    ) {
+        parent::__construct($accessKey, $wsdl, $soapClientOptions, $factory);
+    }
+
+
+    public function getTransactionDetails(
+        string $administrationID,
+        string $GLAccountCode = "",
+        string $StartDate = '2020-01-01',
+        string $EndDate = '2021-01-01',
+        int $financialMode = 1
+    ):array {
+        $arguments = compact("administrationID", "GLAccountCode", "StartDate", "EndDate", "financialMode");
+
+        $response = $this->call('GetTransactionDetails', $arguments);
+        $arrayResponse = $response->GetTransactionDetailsResult->TransactionInfo ?? null;
+
+        if (is_array($arrayResponse)) {
+            return $arrayResponse;
+        }
+
+        UnexpectedTypeException::fromValue($arrayResponse, 'array');

```

Comment:
`throw` is missing here. It was my mistake, it was missing in other places too ;). I have corrected it a couple of days ago.

And of course change it in all places wher it is missing.

```suggestion
        throw UnexpectedTypeException::fromValue($arrayResponse, 'array');
```